demos/notebooks/demo_pipeline.ipynb

```diff
@@ -610,10 +1023,736 @@
+    "Caiman is optimized for parallelization and works well at HPC centers as well as laptops with multiple CPU cores. The cluster is set up with the `setup_cluster()` function, which takes in multiple parameters:\n",
+    "\n",
+    "    c, multiprocessing_pool, n_processes = cm.cluster.setup_cluster(backend='multiprocessing', \n",
+    "                                                     n_processes=None, \n",
+    "                                                     ignore_preexisting=False)\n",
+    "\n",
+    "The `backend` parameter determines the type of cluster used. The default value, `'multiprocessing'`, uses the multiprocessing package, but `ipyparallel` is also available. More information on these choices can be found [here](https://github.com/flatironinstitute/CaImAn/blob/master/docs/CLUSTER.md). You can set the number of processes (cpu cores) to use with the `n_processes` variable: the default value `None` will lead to the function selecting one *less* than the total number of logical cores available.  \n",
+    "\n",
+    "The parameter `ignore_preexisting`, which defaults to `False`, is a failsafe used to avoid overwhelming your resources. If you try to start another cluster, you will get an error. However, sometimes on more powerful machines may want to spin up multiple Caiman environments. In that case, set `ignore_preexisting` to `True` and you will not get an error.\n",
+    "\n",
+    "As discussed in the main notebook, the output variable `multiprocessing_pool` returned by the function is the multicore processing object that will be used in subsequent processing steps. It is a crucial variable for Caiman's parallelization: it is the multicore engine that will be passed around in subsequent stages and is the fulcrum for parallelization. \n",
+    "\n",
+    "The other output that can be useful to check is `n_proceses`, as it will tell you how many processes you successfully set up. \n",
+    "\n",
+    "Once you are done running computations that will use the cluster (motion correction and CNMF), then it can be a useful to save CPU resources by shutting it down: \n",
+    "\n",
+    "    cm.stop_server(dview=multiprocessing_pool)\n",
+    "    \n",
+    "You may also have noticed that we used this method to shut down any pre-existing clusters before starting a new one."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=memmapping></a>\n",
+    "## Appendix 6: Memory mapping\n",
+    "\n",
+    "We mentioned in the body of the notebook that we save the motion corrected file as a memory mapped file. In general, memory mapped files are binary files saved to disk, and the operating system can work with them as if they were in RAM by just loading parts of the files into memory when needed for particular computations. This is known as *out of core computation*. This is how Caiman is able to work with large files without loading them into RAM. \n",
```

Comment:
In general -> ""

Do we need to explain this in the notebook?

Reply:
I think we should explain it _somewhere_: there is this mysterious thing in the first notebook where we just sort of wave toward memory mapping, and the switch from F order to C order.  I want to have it _somewhere_ (though I do think it would be a good idea to push it to docs eventually -- I thought for now an Appendix is good which is basically my "50% chance this goes to docs eventually" bin).

Reply:
I'm fine with killing Appendices, as that is a weird mechanism in general. Listing them here, and thinking that stuff to docs is less essential, stuff to blue boxes is more essential and likely to get issues asked if we remove it:

> 1. logging -- probably to docs
> 2. working with multiple files -- make a blue box probably?
> 3. displaying large files -- blue box?
> 4. estimators and parameters -- to docs
> 5. more on cluster setup and shutdown -- to docs maybe update the page we already have
> 6. memory mapping -- to docs

Reply:
I wonder if we can get away with providing a link to docs. "For guidance on working with large files, see the docs _here_"

Maybe we should have notebooks for working with large files and some of these topics, although I don't know if we'd want to actually distribute big files (can we generate them on-the-fly?) as sample datasets for such future notebooks?

Don't we have a notebook for working with multiple files?

Reply:
That's what I was thinking -- I could replace link to appendix with link to docs, and add some of these appendices to docs (or supplement current docs). For other things just make breakout blue boxes.

Reply:
Sounds good!

Reply:
We have a notebook for multisession registration, but I am not sure if we have a nb specifically tailored for multiple files from a single session. Previous notebook would just extend to that b/c it used `load_movie_chain()` and fed files in a list: I removed that and now reference in that Appendix.

Reply:
I wonder if we might want to split an existing demo movie file in half just to demonstrate multiple files from a single session. Maybe we can explain less if our defaults cover this topic.

Reply:
That's in Appendix 2. But I do **not** want to make this the base -- that is pedagogically just a bad move when this is going to be most people's first exposure focus on the basic logic -- not secondary possible use cases for people with systems that collect data in a certain way.

Reply:
Fair enough.